### PR TITLE
Email with registration launch

### PR DESCRIPTION
**Pull request by dcwalk**

### CODE REVIEW

File: advertising/email-00-registration.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+There are three sliding scale admission levels: Organization Supporter, Supporter, and Reduced Cost. The true cost for us to host this event is $57 per person. Your support at or above that price allows us to offer reduced cost admissions and use sponsorship money elsewhere. If it is difficult to attend even at a reduced cost, contact us at orga@ournetworks.ca. We are able to support a limited number of scholarships and are also looking for volunteers interested in helping at the event in return for complimentary admission. 
+
+https://ournetworks.ca/conference/#registration
+
+⚐ PROGRAM HIGHLIGHTS
+Our call for submissions closed with an overwhelming number of submissions. We’re just finalizing the schedule, but you can expect:
+
+▷ Talks about distributed data stewardship, migrating to peer-to-peer implementations, and the tactics used by existing protocols with a view to changing the internet
+▷ Discussion sessions on transitioning to decentralized projects and imagining that alternatives we want
```
Comment (benhylau) [inline advertising/email-00-registration.md]: >Discussion sessions on transitioning to decentralized projects and imagining **_that_** alternatives we want 

_the_ alternatives?
_what_ alternatives?

File: advertising/email-00-registration.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+Title: Our Networks: registration open + vend at the Yami-ichi ⚑
```
Comment (benhylau) [inline advertising/email-00-registration.md]: I would save this file as `.txt`, not `.md`

Comment (dcwalk) [reply advertising/email-00-registration.md]: Did you see my comment on the previous PR? 

We've been consistently .md until that one .txt, I'm not seeing a strong reason for one over the other as whoever sends the message has a lot of final control on formatting? Raw md is still _unformatted_ but reads nicer when viewed on gh

File: advertising/email-00-registration.md (hunk 1 above)
Comment (dcwalk) [reply advertising/email-00-registration.md]: 👍  I think it was "what"

File: advertising/email-00-registration.md (hunk 2 above)
Comment (benhylau) [reply advertising/email-00-registration.md]: Sorry I missed the one in the previous PR. When it's an .md two lines get folded into one, that's why I prefer keeping as plain text, but I am okay with keeping things consistent as is.

Comment (dcwalk) [reply advertising/email-00-registration.md]: fair point. Lets merge for now and refactor?

**Pull request merged**